crates/telio-nurse/src/aggregator.rs

```diff
@@ -186,7 +192,7 @@ pub struct ConnectivityDataAggregator {
     aggregate_relay_events: bool,
     aggregate_nat_traversal_events: bool,
 
-    wg_interface: Arc<dyn WireGuard>,
+    wg_interface: Weak<dyn WireGuard>,
```

Comment:
Why is this change made? Up until today we avoided using weak pointers in favor of explicitly controlling the order of destruction of various things (eg. `MeshnetEntities::stop`).

Reply:
Well, I thought that it makes sense that aggregator won't keep wg interface alive, but I can remove it explicitely

Reply:
While I think this particular dependency is kinda weak, idk whether that's the case in the other places, so you can add some task if you are convinced, but I'm not entirely convinced that it is a good idea to change it everywhere